CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ and adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
   of code as an error.
 - The `GlobalVariablesChecker` now allows type alias assignment statements at the top level, i.e., doesn't flag such
   lines of code as an error.
+- The HTML Report now displays the error message for C0303 `trailing-whitespace`, and only trailing whitespaces are
```

Comment:
The first part isn't correct, the error message was always being triggered. (If you were having trouble triggering it it's likely due to having your IDE automatically strip trailing whitespace.)

This change also isn't just for the HTML reporter, but the colour reporter as well. So you don't need to specify the reporter type.